stellarhome/src/main/java/com/stellarhome/DTO/UserDTO.java

```diff
@@ -0,0 +1,19 @@
+package com.stellarhome.DTO;
+
+import lombok.*;
+
+@Data
+@Getter
```

Comment:
You should looking for a single annotation to merge all those @Setter,@Getter or simple decide to use @Builder @DilbaniE @DilbaniEB